### DISCUSSION TITLE
How to stop the flooding?

### DISCUSSION THREAD

**Discussion by pneumus** [bluez/bluez#889] [Q&A]
When executing the command "scan on" in the interactive prompt of bluetoothctl, 
it will immediately flood the screen with [NEW] and [CHG] devices.
Even after executing "scan off" it will keep flooding the screen with [DEL] devices.
Using bluetoothctl in a non-interactive mode ( i.e. bluetoothctl --timeout 3 scan on ) can help,
but what if I want to stop the flood in the interactive shell? Is there a way to do that? Thanks!

**Comment by pdl-user** [comment]
In addition to the above, neighbors' bluetooth devices are amplifying the flood. This is overwhelming for someone just getting started with bluetooth.
